mkdocs.yml

```diff
@@ -69,6 +69,8 @@ nav:
           - guides/install-certificate.md
           - guides/clipboard.md
           - guides/touch-id.md
+      - Gestures:
```

Comment:
I'm not sure if this document needs to have its own category - IMO it's fine to put it under Driver Actions.